ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -71,14 +71,17 @@
           </template>
 
           <!-- Civic Address -->
-        <v-row no-gutters class="px-6 pt-1">
-          <v-col cols="3" class="pt-1">
-            <h3>Civic Address</h3>
-          </v-col>
-          <v-col cols="9" class="pt-1">
-            <p>{{ '' || '(Not Entered)' }}</p>
-          </v-col>
-        </v-row>
+          <template>
```

Comment:
A `template` tag is usually just used if you want to conditionally render something. 
I'm not sure it has a purpose here.

Reply:
Removed